examples/DEVELOPER.md

```diff
@@ -109,6 +109,17 @@ responds_without_header \
 
 `responds_without_header` can accept additional curl arguments like `responds_with`
 
+#### Utility functions: `wait_for`
+
+You can wait for some amount of time for a command to return 0
```

Comment:
Can you clarify that the amount of time is in `seconds`, please?

Reply:
updated